common/content/buffer.js

```diff
@@ -1648,7 +1648,7 @@ const Buffer = Module("buffer", {
 
         mappings.add(myModes, ["[["],
             "Follow the link labeled 'prev', 'previous' or '<' if it exists",
-            function (count) { buffer.followDocumentRelationship("previous"); },
+            function (count) { buffer.followDocumentRelationship("prev"); },
```

Comment:
this doesn't actually just look for link tags with a  rel of `previous`.
It actually looks at the option for `previouspattern` and then looks at `link`, `a`, and whatever is in `hinttags` option and see whether it patches the `previouspattern` pattern.